extensions/blocks/recurring-payments/editor.scss

```diff
@@ -1,18 +1,32 @@
 @import './view.scss';
 
-.wp-block-jetpack-membership-button {
-	font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Ubuntu, Cantarell,
-	Helvetica Neue, sans-serif;
+.wp-block-jetpack-recurring-payments {
+	font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
```

Comment:
You can re-use the Gutenberg variable for this font-family:

```
@import '../../shared/styles/gutenberg-variables.scss';

font-family: $default-font;
```

Example here:

https://github.com/Automattic/jetpack/blob/8361c7010843b88776a67909db2369c3661bcdc2/extensions/blocks/simple-payments/editor.scss#L2-L5

Reply:
Done!